docs/UserGuide.md

```diff
@@ -153,30 +153,32 @@ Example: `listpersons` lists all persons in the catalogue to the user.
 Leave the current page and go back to the parent scope (one level up).
 
 Format: `leave`
-- Leaves the current page (clear the right-most non-empty dashboard) and shifts the scoping status to the parent scope.
-- If the app is already in the global, `PERSON_LIST`, or `PROJECT_LIST`, then the command takes no effect.
+- Leaves the current page (clear the right-most non-empty dashboard) and shifts the scoping status to the parent scope
+- If the app is already in the global, `PERSON_LIST`, or `PROJECT_LIST`, then the command takes no effect
 
 # **Features** in project_list scope
 
 ### Start work on an existing project `startproject `
+
 Initialises the project specified.
 
 Format: `startproject (INDEX)`
-- Initialises the project at the specified INDEX.
-- The index refers to the index number shown in the displayed project list.
+- Initialises the project at the specified INDEX
+- The index refers to the index number shown in the displayed project list
```

Comment:
Could you please clarify which point should contains full stop and which should not? Why "List all persons....catalogue." has full stop but then "Leaves the current page......scope" has no?

Reply:
anything in point form does not have full stops, everything else has full stops

Reply:
Make sense.